integration_tests/src/test/kotlin/utils/OutputReportParser.kt

```diff
@@ -16,6 +17,8 @@ val OutputReport.firstTestSuiteOverview: SuiteOverview
 data class OutputReport(
     val args: Any,
     val cost: OutputReportCostNode? = null,
+    @JsonProperty("billable-minutes")
```

Comment:
Does this property definition need to match the `OUTPUT_BILLABLE_MINUTES` value? I see an integration test failure that looks like:
```
integration.MultipleDevicesIT > run tests on multiple devices - android FAILED
    com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "billable_minutes" (class utils.OutputReport), not marked as ignorable (6 known properties: "billable-minutes", "weblinks", "error", "test_results", "args", "cost"])
```